packages/test/test-utils/src/opProcessingController.ts

```diff
@@ -75,13 +80,44 @@ class DeltaManagerMonitor extends DeltaManagerToggle {
     private pendingCount: number = 0;
     private clientId: string | undefined;
     private firstClientSequenceNumber: number = -1;
-    private _latestSequenceNumber: number;
+    private lastOutbound: IDocumentMessage | undefined;
+    private readonly lastInboundPerClient = new Map<string, ISequencedDocumentMessage>();
+    private pendingWriteConnection = false;
+
+    public expectingInboundFrom(outbound: DeltaManagerMonitor): boolean {
```

Comment:
It would be great to have a comment above it describing what it tries to achieve.
Glancing through code, I'm not sure I'm getting all the details...